vpp/clientconfigsrv.go

```diff
@@ -0,0 +1,117 @@
+package vpp
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/satori/go.uuid"
+	"strings"
+)
+
+type ClientContext struct {
+	HostName string `json:"hostname"`
+	Guid     string `json:"guid"`
+}
+
+type Location struct {
+	LocationName string `json:"locationName"`
+	LocationId   int    `json:"locationId"`
+}
+
+type ClientConfigSrvResponse struct {
+	ClientContext      interface{} `json:"clientContext"`
+	AppleId            string      `json:"appleId"`
```

Comment:
a good pattern to keep in mind is to keep initialisms all upper case. `AppleID`,`UID`, `APNToken`, `GUID` etc. 

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
I believe I have address these in the commit that I just pushed